### DISCUSSION TITLE
Default tag list

### DISCUSSION THREAD

**Discussion by Jonrysh** [quodlibet/quodlibet#4228] [Q&A] [answered]
I need to index a large number of completely unindexed files and would like to be able to:

- pop up a default list of tags, say Artist, Title, Performer, Track, etc.
- create tags for all the tracks of an album, the same except for the track number

Is there currently ways to do these things?  If so, how?  If not consider these to be feature requests.

**Comment by declension** [comment, marked as answer]
Have you considered using the musicbrainz lookup plugin? It will fill in all these and more
